src/components/WebexMediaAccess/WebexMediaAccess.jsx

```diff
@@ -70,7 +70,7 @@ export default function WebexMediaAccess({
         size={40}
         onClick={() => adapter.meetingsAdapter[screen.buttonAction](ID)}
         tooltip={screen.buttonTooltip}
-        ariaLabel={screen.buttonHint}
+        ariaLabel={`${screen.buttonLabel}. ${screen.buttonHint}`}
```

Comment:
```js
buttonHint: 'Proceed without camera. This setting cannot be changed once the meeting starts.',

Reply:
the idea is to put the full text in the buttonHint and not do any concatenation here